File: Python/Iniciante/1036.py
import math

a, b, c = input().split()

a = float(a)
b = float(b)
c = float(c)

delta = pow(b, 2) - (4 * a * c)

if delta < 0: print("Impossivel calcular")
else:
  try:
    
    r1 = (-b + math.sqrt(delta)) / (2 * a)
    r2 = (-b - math.sqrt(delta)) / (2 * a)
    print(f"R1 = {r1:.5f}\nR2 = {r2:.5f}")
    
  except ZeroDivisionError: print("Impossivel calcular")

File: Java/Iniciante/1036.java
import java.io.IOException;
import java.util.Scanner;

public class Main {

  public static void main(String[] args) throws IOException {
    Scanner input = new Scanner(System.in);

    double a = input.nextDouble();
    double b = input.nextDouble();
    double c = input.nextDouble();
    double x1 = 0;
    double x2 = 0;

    double delta = Math.pow(b, 2) - (4 * a * c);

    if(delta > 0 && a > 0 && b > 0){
      x1 = ((b * -1) + Math.sqrt(delta)) / (2 * a);
      x2 = ((b * -1) - Math.sqrt(delta)) / (2 * a);

      System.out.printf("R1 = %.5f\n", x1);
      System.out.printf("R2 = %.5f\n", x2);
    }
    else{
      System.out.println("Impossivel calcular");
    }
  }
}
